pkgs/development/python-modules/algopy/default.nix

```
@@ -0,0 +1,37 @@
+{ buildPythonPackage
+, lib
+, fetchPypi
+, numpy
+, scipy
+, python
+, nose
+}:
+
+buildPythonPackage rec {
+  pname = "algopy";
+  version = "0.5.7";
+
+  src = fetchPypi {
+    inherit pname version;
+    extension = "zip";
+    sha256 = "1pxphhajk6b6kd0ich6jfwrvjv3rw7vz7dswb2iqz1g3zivgcmb9";
+  };
+
+  checkInputs = [ python nose ];
```

Why do we need nose?

```suggestion
  checkInputs = [ nose ];
```

---

Why not `checkInputs = [ pytestCheckHook ]`?